### PR TITLE
Adds Gungans for the purpose of conflicting with dismemberment

### PR DESCRIPTION
**Pull request by KazeEspada**
Glorfs are an amphibious annoying humanoid with long dangly ear flaps. Luckily they all look the same, and NT only allows them to be assistants.

Sprites by Nienhaus
glorf

:cl:KazeEspada
add:Glorfs are a new race that are playable. The exact opposite of special snowflake. You can only customize their eye color.
/:cl:

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
thumbs down

**Comment by Jazaen** [comment]
Ehh, not a really good plan. And if you really do want to go through with this, make sure that, instead of "JarJar says" it's "JarJar babbles", so silicons can tell them apart from humans.

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -719,3 +719,29 @@ var/global/list/synth_flesh_disguises = list()
...
 		return ..()
+
+/*
+Glorfs - Gungan lookalikes
+*/
+
+/datum/species/glorf
+	name = "Glorf"
+	id = "glorf"
+	say_mod = "yammers"
```
Comment (KazeEspada) [inline code/modules/mob/living/carbon/human/species_types.dm]: They yammer.

**Comment by darkpaladin109** [comment]
Doesn't sound terrible, I guess.

**Comment by GunHog** [comment]
Eh. Okay, I guess. :+1: Filthy assistant scum.

**Comment by KorPhaeron** [comment]
I would close this for being a lazy joke/unsubtle reference that doesnt fit at all but that would make me a hypocrite.

I hope someone else does though.

File: config/game_options.txt [hunk 2]
```diff
@@ -269,6 +269,7 @@ ROUNDSTART_RACES lizard
 ROUNDSTART_RACES plasmaman
 #ROUNDSTART_RACES shadow
 #ROUNDSTART_RACES shadowling
+ROUNDSTART_RACES glorf
```
Comment (KorPhaeron) [inline config/game_options.txt]: >on by default

**Comment by WJohn** [comment]
Not like this... not like this!

File: code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -383,6 +387,8 @@
 					S = ears_list[H.dna.features["ears"]]
 				if("body_markings")
 					S = body_markings_list[H.dna.features["body_markings"]]
+				if("glorfears")
+					S = glorfears_list[H.dna.features["glorfears"]]
```
Comment (KorPhaeron) [inline code/modules/mob/living/carbon/human/species.dm]: Is there not a less snowflake way to do this?

**Comment by Incoming5643** [comment]
:-1: Bombad

File: code/modules/mob/living/carbon/human/species.dm (hunk 3 above)
Comment (KazeEspada) [inline code/modules/mob/living/carbon/human/species.dm]: Nope, whoever wrote the human renderer made it super static.

**Comment by TrustyGun** [comment]
If you actually go through with this, change their skin color to something else so they are more distinguishable from humans.

**Comment by KazeEspada** [comment]
That's the sprite color? Also, ears. Yammers, and Me-sa, you-sa.

**Comment by KazeEspada** [comment]
Come on guys, this was a bitch and a half to code.

**Comment by KorPhaeron** [comment]
Well I did tell you days ago I would deny it, and never enable it on server if it got in. I tried to save you the time.

**Comment by Shadowlight213** [comment]
:+1: add a button/config option that auto swaps lizard players to this.

**Comment by ATDoop** [comment]
they should never be able to finish a sentence and always GLORF half way through

**Comment by KazeEspada** [comment]
Shadowlight... that's called disabling lizards and enabling glorfs.

**Comment by PKPenguin321** [comment]
now all we need is the hedgehog-man race

**Comment by Iamgoofball** [comment]
:-1: WE DONT NEED MORE FUCKING SPECIAL SNOWFLAKE FURRY RACES

**Comment by KazeEspada** [comment]
1. Glorfs are slimy, not furry.
2. How is it special snowflake when every glorf is nearly identical?

**Comment by octareenroon91** [comment]
This bullshit.

I like this bullshit.

**Comment by Aloraydrel** [comment]
r e d d i t

**Comment by Nienhaus** [comment]
:+1:

**Comment by SweaterKittens** [comment]
Good god no, please no more joke races. Lizards are already a joke, and Flypeople even more so. If we're adding more races make them more than a meme.

**Comment by Shadowlight213** [comment]
@sweaterkittens tajarans?

**Comment by KazeEspada** [comment]
Tajarans are "too furry".

**Comment by CosmicScientist** [comment]
:-1: tgstation is better than Star Wars.

**Comment by Incomptinence** [comment]
It's terrible 
It's an abomination 

I love it

**Comment by Dorsisdwarf** [comment]
Add a new job, just for Glorfs

The job spawns in maint.

Glorfs are valid kills.


Then we'll all be happy

**Comment by demonfiren** [comment]
This is shit, but at least it's hilarious shit.

**Comment by ike709** [comment]
:+1: /tg/ Wars: Coderbus Strikes Back

**Comment by Dorsisdwarf** [comment]
I just realised these don't go glorf when you beat them.
:-1: :-1: :-1: :-1: :-1: :-1:

**Comment by Steelpoint** [comment]
>actually adding Jar Jar Binks to the game.

The only role I would accept these abominations in is a round start maintenance dweller that spawns naked.

**Comment by KazeEspada** [comment]
Isn't that an assistant?

**Comment by KorPhaeron** [comment]
Nevermind I'll be a hypocrite after all.

If any other maintainers actually think this is a good idea feel free to reopen it.

**Comment by duncathan** [comment]
Psyke

**Comment by tedward1337** [comment]
GLORF

**Comment by tedward1337** [comment]
rip

**Pull request closed without merging**